web-modules/pom.xml

```diff
@@ -25,12 +25,12 @@
         <module>javax-servlets-2</module>
         <module>jee-7</module>
         <module>jooby</module>
-        <module>linkrest</module>
-        <module>ninja</module>-->
+        <module>linkrest</module>-->
+        <module>ninja</module>
         <!-- <module>play-modules</module> --> <!-- Not a maven project -->
         <!-- <module>raml-modules</module> --> <!-- Not a maven project -->
-        <!--<module>ratpack</module>
-        <module>resteasy</module>
+        <module>ratpack</module>
+        <!--<module>resteasy</module>
         <module>restx</module>
```

Comment:
we need to define jooby and restx in the submodule fashion in the root pom.xml under *first and *second profile respectively so they are still executed in the jdk8 builds.

Reply:
done